packages/transport-webrtc/src/private-to-private/signaling-stream-handler.ts

```diff
@@ -96,11 +87,6 @@ export async function handleIncomingStream ({ peerConnection, stream, signal, co
       signal,
       log
     })
```

Comment:
should we still `log.trace('recipient connected, closing signaling stream')` ?

Reply:
I would have thought not, since this function isn't closing the signalling stream any more.

Reply:
I was thinking that we're still waiting for `readCandidatesUntilConnected` and the last log was "reading until connected", so in logs it could look like you're never done reading, nor connected, unless we see a "done reading" type message"

maybe my original comment should have asked for keeping `log.trace('recipient connected, stopped reading')`